client/src/components/MediaImage.jsx

```diff
@@ -25,12 +25,21 @@ class MediaImage extends React.Component {
 
     return (
       <div className="TP-mediaImage">
-          <canvas id="canvas" width="1000" height="200"></canvas>
+        <canvas id="canvas" class="canvas" width="1000" height="200"></canvas>
         <div className="TP-currentTime">{`${this.props.currentTime}`}</div>
         <div className="TP-duration">{`${this.props.duration}`}</div>
         <div className="TP-commentBlock">
           {comments}
         </div>
+        <input
+          id="TP-timeline"
```

Comment:
Something seems a little off here - the "TP-xxxx" pattern is used as both id and className, I would be careful to use consistent naming conventions to avoid confusion later down the line.